edumap/test/models/lesson_test.rb

```diff
@@ -86,4 +86,3 @@ class LessonTestSorting < ActiveSupport::TestCase
     assert_equal (Lesson.sorted_by :created_at_bob), [first, second]
   end
 end
-
```

Comment:
Added another test case where the lesson has a range of levels (i.e., if it specifies 1 and 3, will match query for grade 2)